recipe/build.sh

```diff
@@ -41,7 +40,11 @@ export BAZEL="$(pwd)/../bazel-standalone"
     ${EXTRA_BAZEL_ARGS:-} \
     --cpu=${TARGET_CPU} \
     --local_cpu_resources=${CPU_COUNT} \
+    --spawn_strategy=standalone \
     //python/dist:binary_wheel \
     --define=use_fast_cpp_protos=true
 
```

Comment:
Is it worth looking at running tests? It doesn't appear to be running them now, from the code at least.